homeassistant/components/shelly/update.py

```diff
@@ -256,11 +257,12 @@ def _ota_progress_callback(self, event: dict[str, Any]) -> None:
         if self.in_progress is not False:
```

Comment:
This check is a bit weird to me, I think it means Shelly update entities won't show an update in progress if the update was initiated outside of Home Assistant?

Reply:
I think you are correct, I looked at the PR history and didn't find why it was added and it looks like if the update is triggered outside HA we will skip the progress status.

Reply:
Should that be fixed in a separate PR?

Reply:
This was intentional. I think we shouldn't show the update progress if the update was started from outside. I don't know how other integrations approach this.

Reply:
Why not ? I thin kwe should always reflect the real status of the device

Reply:
Maybe you are right. I think we can change this in the separate PR.